playwright/sync_api/_context_manager.py

```diff
@@ -37,6 +37,7 @@ def __enter__(self) -> SyncPlaywright:
             loop = asyncio.get_running_loop()
         except RuntimeError:
             loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(loop)
```

Comment:
It is required because the [loop parameter in asyncio is deprecated](https://docs.python.org/3/library/asyncio-sync.html#asyncio.Event) and this set up the loop correctly when there is no event loop in the current thread correctly